conda-store-server/conda_store_server/_internal/server/templates/conda-store-ui.html

```diff
@@ -22,7 +22,7 @@
              REACT_APP_AUTH_TOKEN: "",
              REACT_APP_STYLE_TYPE: "green-accent",
              REACT_APP_SHOW_LOGIN_ICON: "true",
-             REACT_APP_API_URL: "{{ url_for('redirect_root_to_ui') }}",
+             REACT_APP_API_URL: "{{ url_for('redirect_root_to_ui') }}/",
```

Comment:
I'm not sure if there is a better way to do this, like maybe on the ui side? The rest of the interactions between the ui and api seem to work properly with and without this change.